### ISSUE TITLE
Icons added by :before pseudoclasses are inaccessible to screenreaders

### ISSUE THREAD

**Issue by samsworldofno** [coopdigital/coop-frontend-toolkit#31]
Currently, classes like  `.message-prompt` add icons using a `:before` pseudoclass. The add this element as plain text, such as `content: '\e607'`.

![screen shot 2016-11-14 at 10 14 27](https://cloud.githubusercontent.com/assets/61320/20260771/0798f156-aa53-11e6-967b-4ea40031ad92.png)

Screenreaders recognise this content but are unable to announce it. Our solution was to remove the icon from the `*-prompt` class, and adding it on the page as a separate `<span>` element with `aria-hidden="true"`. This will enable the icon to remain visible, but it will be hidden from screen reader users. 

Is there a better solution?

Our Jira issue: https://co-op-digital.atlassian.net/browse/MMDT-727
